mne_bids/write.py

```diff
@@ -933,6 +1016,38 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
         bids_raw_folder = bids_fname.split('.')[0]
         bids_fname = op.join(bids_raw_folder, bids_fname)
 
+    # Anonymize
+    if anonymize is not None:
+        if 'daysback' not in anonymize:
+            raise ValueError('`daysback` argument required to anonymize.')
+        daysback = anonymize['daysback']
+        daysback += 36525  # add 100 years (or 36525 days) back by default
```

Comment:
what if someone provides `daysback=None` as in MNE? It will crash with an obscure error.